README.md

```diff
@@ -56,6 +56,80 @@ Since this library is used by Home Assistant, see <https://www.home-assistant.io
 
 So follow that advice and try to scrape all fields at once, similar to the `get_form_action_url_and_hidden_inputs` in helpers.py.
 
+## Example Usage - CLI
+
+The opower library comes with a CLI interface to get familiar with the library and read actual data from your utility.
+
+```sh
+# Show information on how to call the library CLI interface
+python -m opower --help
+
+# Example: show recent usage only, for all accounts:
+python -m opower --utility comed --username <username> --password <password> --usage_only --aggregate_type day
+[...]
+start_time	end_time	consumption	start_minus_prev_end	end_minus_prev_end
```

Comment:
I've been meaning to remove the last 2 columns. I only added them during development when I was fixing an issue with daylight savings time. I've only kept just in case. Especially because I might have to migrate to new API that uses graphql at some point. Anyway I'd rather not publish these in the README. I think it would be better to remove the output.